modules/passkey/contracts/4337/README.md

```diff
@@ -0,0 +1,102 @@
+# Passkey 4337 Support
```

Comment:
Moved from `experimental` folder and deleted the mention to the `SafeSignerLaunchpad`. Otherwise the file remained mostly unchanged.